src/components/asset-list/RecyclerAssetList2/LegacyWrappedTokenFamilyHeader.tsx

```diff
@@ -0,0 +1,211 @@
+import { ButtonPressAnimation } from '@/components/animations';
+import TokenFamilyHeaderIcon from '@/components/token-family/TokenFamilyHeaderIcon';
+import { Text } from '@/design-system';
+import { ImgixImage } from '@/components/images';
+import { ThemeContextProps } from '@/theme';
+import { useOpenCollectionsStore } from '@/state/nfts/openCollectionsStore';
+
+type Props = {
+  name: string;
+  total?: number;
+  image?: string;
+  theme: ThemeContextProps;
+  testID?: string;
+};
+
+export default React.memo(function WrappedTokenFamilyHeader({ name, total, image, theme, testID }: Props) {
+  const { nfts_enabled } = useRemoteConfig();
+  const nftsEnabled = useExperimentalFlag(NFTS_ENABLED) || nfts_enabled;
+
+  const nameKey = `${name}-legacy`;
+
+  const isFamilyOpen = useOpenCollectionsStore(state => state.isCollectionOpen(nameKey));
+
+  const handleToggle = useLatestCallback(() => useOpenCollectionsStore.getState().toggleCollection(nameKey));
+
+  if (!nftsEnabled) return null;
+
+  return (
+    <TokenFamilyHeader
+      childrenAmount={total}
+      familyImage={image}
+      isOpen={isFamilyOpen}
+      onPress={handleToggle}
+      testID={testID}
+      theme={theme}
+      title={name}
+    />
+  );
+});
+
+export const TokenFamilyHeaderAnimationDuration = 200;
+export const TokenFamilyHeaderHeight = 50;
+
+const sx = StyleSheet.create({
+  amountContainer: {
+    marginRight: 10,
+  },
+  center: {
+    alignItems: 'center',
+    flexDirection: 'row',
+    justifyContent: 'center',
+  },
+  chevron: {
+    height: 18,
+    width: 8,
+  },
+  content: {
+    alignItems: 'center',
+    flexDirection: 'row',
+    height: TokenFamilyHeaderHeight,
+    justifyContent: 'space-between',
+    padding: 19,
+    paddingRight: 14,
+    width: '100%',
+  },
+  title: {
+    flex: 1,
+    paddingRight: 9,
+  },
+});
+
+type TokenFamilyHeaderProps = {
+  childrenAmount?: number;
+  emoji?: string;
+  familyImage?: string;
+  isOpen?: boolean;
+  onPress?: () => void;
+  testID?: string;
+  title: string;
+  theme: ThemeContextProps;
+};
+
+const TokenFamilyHeader = ({ childrenAmount, emoji, familyImage, isOpen, onPress, testID, title, theme }: TokenFamilyHeaderProps) => {
+  const { colors } = theme;
+
+  const toValue = Number(!!isOpen);
+
+  const aRef = useRef<{
+    animation: Animated.Value;
+    toValue: number;
+    isRunning: boolean;
+  } | null>(null);
+
+  // HACK ALERT
+  // we are reusing the same animated value across rows when we recycle them
+  // only happens when previous and next row are both open or closed
+  // but also we don't want the animated value to change
+  // during the animation too
+  if (aRef.current === null || (aRef.current.toValue !== toValue && !aRef.current.isRunning)) {
+    aRef.current = {
+      animation: new Animated.Value(toValue),
+      isRunning: false,
+      toValue,
+    };
+  }
+
+  const { animation } = aRef.current!;
+
+  const handlePress = useLatestCallback(() => {
```

Comment:
we don't need to memo in this component generally, any time a component takes children memoizing the props is of no value as every time children changes (every render) the props change too (because children is a new object).

nice thing with react compiler is it does memo children, but then you dont need useMemo either